pkg/stanza/fileconsumer/attributes.go

```diff
@@ -36,7 +37,15 @@ func (f *FileAttributes) HeaderAttributesCopy() map[string]any {
 // resolveFileAttributes resolves file attributes
 // and sets it to empty string in case of error
 func resolveFileAttributes(path string) (*FileAttributes, error) {
-	resolved, symErr := filepath.EvalSymlinks(path)
+	resolved := ""
+	var symErr error
+	// Dirty solution, waiting for this permanent fix https://github.com/golang/go/issues/39786
+	// EvalSymlinks on windows is partially working depending on the way you use Symlinks and Junctions
```

Comment:
Should we try to evaluate the link, and fall back to `resolved = path` only if it fails?

Reply:
Hum.. You mean a hacky test like `if (error == "too many links") { resolved = path }` ?
Why not. Anyway nothing is perfect here, so that can be a good compromise until it avoid tons of stacktraces and missing attributes.

If you confirm, I can change the condition.

Reply:
I was thinking even if we could check for any error, that's better than always setting `resolved = path`, but yeah, if you think we can check for a specific error, that's even better in my opinion.

Reply:
I changed the logic, I'm not sure at all with my Golang level if it's the proper way to compare an "error" but that seems.

But additionaly, I'm wondering if it will not cause performance issues to test `EvalSymlinks` regarding to the code here https://github.com/golang/go/blob/e09bbaec69a8ff960110e13eabb3bef5331ecb0c/src/path/filepath/symlink.go#L98-L101

What do you think about that ?

Reply:
If it's going to drill through 255 links, maybe it is just best to skip it always on windows. I'm fine with it.

Reply:
OK, It's rolled back to the initial approach